src/devtools/client/debugger/src/components/SecondaryPanes/Frames/NewFrames.tsx

```diff
@@ -46,13 +44,16 @@ function FramesRenderer({
   const replayClient = useContext(ReplayClientContext);
   const sourcesState = useAppSelector(state => state.sources);
   const { rangeForSuspense: focusWindow } = useContext(FocusContext);
-  const { endpoint } = useContext(SessionContext);
   const dispatch = useAppDispatch();
 
+  if (focusWindow === null) {
+    return null;
+  }
```

Comment:
Returning `null` here still feels odd. I guess this will only ever happy briefly during initialization so maybe it doesn't matter.